packages/components/src/Modal/GenericModal/subcomponents/ModalBody/ModalBody.tsx

```diff
@@ -1,17 +1,17 @@
 import React from "react"
 import classnames from "classnames"
-import styles from "./ModalSection.module.scss"
+import styles from "./ModalBody.module.scss"
 
-export type ModalSectionProps = {
+export type ModalBodyProps = {
   inputEdit?: boolean
   children: React.ReactNode
 }
 
-export const ModalSection = ({
+export const ModalBody = ({
```

Comment:
This looks like a new rename 😛 I think `ModalSection` was fine (given the old name was `GenericModalSection`).

I'm not too fussed about it, except that it's kinda weird seeing `ModalHeader` and `ModalFooter` use `ModalBody` 😬